config/selectionCfg_MuTau_ttCR_Legacy2017.cfg

```diff
@@ -13,11 +13,11 @@ invEllypsMassCut2 = ((tauH_SVFIT_mass-129.)*(tauH_SVFIT_mass-129.))/(53.*53.) +
 # ABCD regions used in the analysis - DeepTau
 SR           = isOS != 0 && dau1_iso < 0.15 && dau2_deepTauVsJet >= 5                           # signal region: opposite sign, isolated taus  
 SStight      = isOS == 0 && dau1_iso < 0.15 && dau2_deepTauVsJet >= 5			                # B region				      
-OSinviso     = isOS != 0 && dau1_iso < 0.15 && dau2_deepTauVsJet >= 2 && dau2_deepTauVsJet < 5  # C region				      
-SSinviso     = isOS == 0 && dau1_iso < 0.15 && dau2_deepTauVsJet >= 2 && dau2_deepTauVsJet < 5  # D region				      
+OSinviso     = isOS != 0 && dau1_iso < 0.15 && dau2_deepTauVsJet >= 1 && dau2_deepTauVsJet < 5  # C region				      
+SSinviso     = isOS == 0 && dau1_iso < 0.15 && dau2_deepTauVsJet >= 1 && dau2_deepTauVsJet < 5  # D region				      
 
 # CR ttbar
-ttCR_invMcut = baseline , btagMM, isBoosted != 1 , invEllypsMassCut2, !(isVBF == 1 && VBFjj_mass > 500 && VBFjj_deltaEta > 3 && ((bjet1_bID_deepFlavor > 0.3093) || (bjet2_bID_deepFlavor > 0.3093)))
+ttCR_invMcut = baseline , btagMM, isBoosted != 1 , invEllypsMassCut2, !(isVBF == 1 && VBFjj_mass > 500 && VBFjj_deltaEta > 3 && (((dau1_pt > 25 && dau2_pt > 25 && (dau1_pt <= 40 || dau2_pt <= 40)) && VBFjj_mass > 800 && VBFjet1_pt > 140 && VBFjet2_pt > 60) || isVBFtrigger==0) && (bjet1_bID_deepFlavor > 0.3033 || bjet2_bID_deepFlavor > 0.3033) )
```

Comment:
Here the change in the bTag WP is ok, but the VBF part should not be changed because this is MuTau, not TauTau

Reply:
Yes, but the WP should be 0.3033 and not 0.3093 right?

Reply:
correct for 2017 the btag WP is 0.3033